recipes/soprano/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "Soprano" %}
+{% set version = "0.8.13b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CCP-NC/soprano/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e1d8ab62c4d410ab1052021d7ae645296fec2bb8d63938014584c6aa7038961b
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Need to list the `console_script` entry points:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  entry_points:
    - vasp2cell = soprano.scripts.vasp2cell:__main__
```
etc.

---

Don't forget to add the 3 others listed in `setup.py`!

---

Yes, did it in a separate commit, thanks!